src/xcode/ENA/ENA/Source/Client/HTTP Client/SAPDownloadedPackage.swift

```diff
@@ -40,8 +42,12 @@ struct SAPDownloadedPackage {
 	
 	// MARK: Properties
 
+	/// The zipped  key package
 	let bin: Data
+	/// The file-verification signature
 	let signature: Data
+	/// The SHA256 tring of the package `bin`
```

Comment:
Is "tring" a thing?

Reply:
Let there be more **S'** in the world

```suggestion
	/// The SHA256 string of the package `bin`
```